English.gemspec

```diff
@@ -19,4 +19,6 @@ Gem::Specification.new do |spec|
     `git ls-files -z 2>#{IO::NULL}`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
   spec.require_paths = ["lib"]
+  spec.bindir        = "exe"
```

Comment:
Can you remove this and next line? These configuration is for distribute executables for gem users. `exe/alias_markdown_table_generator` seems for developer utility, not `English` users.

Reply:
Yes, thank you, that's a great point. I have removed those lines.